### PR TITLE
WT-10814 Avoid RTS directly unpacking cells

### PR DESCRIPTION
**Pull request by raviprakashgiri29**
Added a couple of helper functions `__wt_cell_get_ta ` and `__wt_cell_get_tw ` to retrieve the time aggregate, and time window information from the cell and avoid RTS accessing cell content. [WT-11629](https://jira.mongodb.org/browse/WT-11629) is a follow-up ticket to remove other cell unpacking functions from RTS.

### CODE REVIEW

File: src/include/cell_inline.h [hunk 1]
```diff
@@ -1226,6 +1226,26 @@ __wt_cell_unpack_kv(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CELL
     __cell_unpack_window_cleanup(session, dsk, NULL, unpack_value);
 }
 
+/*
+ * __wt_cell_get_ta --
+ *     Set the time aggregate from an unpacked address cell.
```
Comment (kommiharibabu) [inline src/include/cell_inline.h:1231]: How about the comment something like `Get the underlying time aggregate from ...`. The same for the next function.

**Review by korteland — APPROVED** [review]
I agree with Hari's comment, LGTM otherwise. Thanks for picking this up.

File: src/include/cell_inline.h (hunk 1 above)
Comment (raviprakashgiri29) [reply src/include/cell_inline.h]: Fixed in 3ddd3c4e38653a29b7110261c7695b473975cf75

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**